samples/sensor/i3g4250d/src/main.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2021 Jonathan Hahn
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+
+#define SAMPLING_INTERVAL_MS 5
+
+static int set_sampling_frequency(const struct device *sensor, double sampling_frequency)
+{
+	struct sensor_value setting = {};
+
+	sensor_value_from_double(&setting, sampling_frequency);
+
+	return sensor_attr_set(sensor,
+			       SENSOR_CHAN_GYRO_XYZ, SENSOR_ATTR_SAMPLING_FREQUENCY, &setting);
+}
+
+static void fetch_and_display(const struct device *sensor)
+{
+	struct sensor_value gyro[3];
+
+	int rc = sensor_sample_fetch(sensor);
+
+	if (rc != 0) {
+		printf("ERROR: Failed fetching gyro values: %d.\n", rc);
+	}
+	rc = sensor_channel_get(sensor,
+				SENSOR_CHAN_GYRO_XYZ,
+				gyro);
+
+	if (rc != 0) {
+		printf("ERROR: Failed getting gyro values: %d\n", rc);
```

Comment:
same here, random data will likely be printed afterwards.

Reply:
added a `return`statement here as well